apps/workbench-testing-app/src/app/popup-opener-page/popup-opener-page.component.ts

```diff
@@ -115,7 +115,7 @@ export class PopupOpenerPageComponent implements OnDestroy, AfterViewInit {
     this._coordinateAnchor$ = defer(() => this.form.get(ANCHOR)
       .valueChanges
       .pipe(
-        startWith(this.form.get(ANCHOR).value as object),
+        startWith(this.form.get(ANCHOR).value as Record<string, unknown>),
```

Comment:
I suggest removing this cast as not intuitive. I am aware that removing the cast will cause a deprecation warning, which, however, is a false-positive. What do you think?

Reply:
As an alternative we would create an `AnchorFormGroup` interface to avoid the deprecation warning:
```
interface AnchorFormGroup {
  x: string;
  y: string;
  width: string;
  height: string;
}

...

startWith(this.form.get(ANCHOR).value as AnchorFormGroup),
map(anchorFormValue => ({
    x: Number(anchorFormValue.x),
    y: Number(anchorFormValue.y),
    width: Number(anchorFormValue.width),
    height: Number(anchorFormValue.height),
  }),
),
```

Reply:
Introduced interface